src/index.ts

```diff
@@ -27,6 +27,7 @@ program
   .action(action(program))
   .option("-s, --shell <shell>", `shell to use for command execution, supported shells: ${supportedShells}`)
   .option("-c, --check", `check if shell is in an inshellisense session`)
+  .option("-p, --parent-term-exit", `if the terminal with inshellisense is closed, exit the parent terminal`)
```

Comment:
```suggestion
  .option("--parent-term-exit", `when inshellisense is closed, kill the parent process`)
```
Can we update it to not have a short flag and remove anything about killing the parent terminal (a user could have a parent shell or some other parent process)

Reply:
Removed!